pystiche/papers/ulyanov_et_al_2016/loss.py

```diff
@@ -0,0 +1,172 @@
+from typing import Union, Optional, Sequence, Any, Dict
+from collections import OrderedDict
+import torch
+from pystiche.enc import Encoder, MultiLayerEncoder
+from pystiche.ops import (
+    MSEEncodingOperator,
+    GramOperator,
+    MultiLayerEncodingOperator,
+)
+from pystiche.image import extract_batch_size
+from pystiche.loss import MultiOperatorLoss
+from .utils import ulyanov_et_al_2016_multi_layer_encoder
+
+
+class UlyanovEtAl2016MSEEncodingOperator(MSEEncodingOperator):
+    def __init__(
+        self, encoder: Encoder, score_weight: float = 1e0, impl_params: bool = True,
+    ):
+        super().__init__(encoder, score_weight=score_weight)
+        self.double_batch_size_mean = impl_params
+
+
+    def calculate_score(self, input_repr, target_repr, ctx):
+        score = super().calculate_score(input_repr, target_repr, ctx=ctx)
```

Comment:
Is there a reason you call with a keyword argument `ctx=ctx`? If not please remove.

Reply:
This is a required positional argument in `calculate_score()` in the class `GramOperator`. Without this a `TypeError` is created. However, this is not used there. Should this be removed in the class `GramOperator`?

Reply:
You misunderstood: I want to know why you pass it as keyword argument, i.e. `calculate_score(..., ctx=ctx)` instead as positional `calculate_score(..., ctx)`. Even if we dont use it `GramOperator`, it has to be there to be in compliance with the [base method](https://github.com/pmeier/pystiche/blob/cad5ab6e9485680f2543cf4397d0d21e72a88b9e/pystiche/ops/op.py#L228-L235).